### PR TITLE
Fixing demographics fields 

### LINKED ISSUE #1016: Fix the ordering of Demographics
The demographics section fields are out of order from the final design

https://www.figma.com/file/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?type=design&node-id=435-671&mode=design&t=S6kvOiZl3UJ6ypyy-0

### PR DESCRIPTION
**Pull request by KennethSkylight**
# PULL REQUEST

## Summary
The Demographics data ordering was out of place and is missing some fields. The PR fixes these issues. **Please ignore the blank fields. They will be replaced with "N/A" in another PR**

## Related Issue
Fixes #1016

Screenshots:

image

### CODE REVIEW

**Review by BobanL — DISMISSED** [review]
looks good!

**Comment by sarahtress** [comment]
@KennethSkylight it looks gtm except for 2 things:

- should be "Patient IDs" not "Patient ID"
- is County a field that's available to show? I thought it was, but can remove from the design if it's not

**Comment by KennethSkylight** [comment]
@sarahtress I don't see county in any of our test data

Here's the updated screenshot
image

**Comment by sarahtress** [comment]
> @sarahtress I don't see county in any of our test data
> 
> Here's the updated screenshot image

hmm interesting, I'm seeing county right at the top of this RR. is that an anomaly for this one example or are you able to see that in the other RR samples?
Screenshot 2024-01-10 at 10 56 44 AM

everything else looks good though!

**Comment by KennethSkylight** [comment]
@sarahtress I see. I think it might be getting lost in the fhir converter? I'll add the field and we'll have to find that missing field later.

image

**Comment by sarahtress** [comment]
> @sarahtress I see. I think it might be getting lost in the fhir converter? I'll add the field and we'll have to find that missing field later.
> 
> image

okay! sounds good to me, thanks!

File: containers/ecr-viewer/src/app/view-data/components/Demographics.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ interface DemographicsProps {
 }
 
 const Demographics = ({ fhirPathMappings, fhirBundle }: DemographicsProps) => {
+  console.log(evaluate(fhirBundle, fhirPathMappings.patientTribalAffiliation));
```
Comment (JNygaard-Skylight) [inline containers/ecr-viewer/src/app/view-data/components/Demographics.tsx:16]: Remove console log.

**Review by JNygaard-Skylight — CHANGES_REQUESTED** [review]
LGTM, just need to remove a stray `console.log`

File: containers/ecr-viewer/src/app/view-data/components/Demographics.tsx (hunk 1 above)
Comment (KennethSkylight) [reply containers/ecr-viewer/src/app/view-data/components/Demographics.tsx:16]: Done

**Review by JNygaard-Skylight — APPROVED** [review]

**Pull request merged**